README.md

```diff
@@ -207,15 +178,9 @@ Example:
 
```

Comment:
- [ ] Also your getting started section has no instructions as to how one can use this project or application please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/214375641-42c8d02d-9bf5-4d19-8244-ff5bdeadcfae.png)

I suggest please add the relevant instructions and also remove the placeholder _"Describe how a new developer could make use of your project."_ from your Readme.